src/tests/accomodation/accommodation.test.js

```diff
@@ -2,7 +2,14 @@ import chai, { expect } from 'chai';
 import chaiHttp from 'chai-http';
 import faker from 'faker';
 import app from '../../app';
-import accommodation from '../fixtures/accommodation.fixture';
+import accommodation, {
+  createNewBooking,
+  newBooking,
```

Comment:
@higustave-ops Any reason why this PR contains changes which are not relevant to its story?